rules/community/cloudtrail/cloudtrail_aws_config.py

```diff
@@ -0,0 +1,46 @@
+"""Alert on AWS Config"""
+from rules.matchers.matchers import ConfigMatcher
+from streamalert.shared.rule import rule
+
+
+# Populate this list to alert on specific Config Rules, otherwise all rules will be in-scope
+RULES_TO_ALERT_ON = []
```

Comment:
Segue, but this would be a great place to use LookupTables. We would use it internally here, but sadly on the open source repo, LookupTables is an "optional" feature so this wouldn't really work out-of-the-box.

Reply:
No changes requested, just an observation

Reply:
Very useful for sure. Just a way of notifying the user of this possibility

Reply:
I could leave a comment saying could use lookup tables as a hint to the user?